hack/update-vendor.sh

```diff
@@ -318,8 +337,8 @@ for repo in $(tsort "${TMP_DIR}/tidy_deps.txt"); do
     # disallow transitive dependencies on k8s.io/kubernetes
     loopback_deps=()
     kube::util::read-array loopback_deps < <(go list all 2>/dev/null | grep k8s.io/kubernetes/ || true)
-    if [[ -n ${loopback_deps[*]:+"${loopback_deps[*]}"} ]]; then
-      kube::log::error "Disallowed ${repo} -> k8s.io/kubernetes dependencies exist via the following imports: $(go mod why "${loopback_deps[@]}")" >&22 2>&1
+    if (( "${#loopback_deps[@]}" > 0 )); then
```

Comment:
did you make sure this actually fires if there are loopback deps? I can't eyeball this to tell